ilc/client.js

```diff
@@ -14,7 +14,10 @@ import AsyncBootUp from './client/AsyncBootUp';
 import IlcAppSdk from 'ilc-sdk/app';
 import I18n from './client/i18n';
 import {makeAppId} from './common/utils';
+import UrlProcessor from './common/UrlProcessor';
```

Comment:
For some reason I don't see this file in PR

Reply:
I have created this file previously when I worked on leading trailing slash task